compiler/test/runnable/issue18919.d

```diff
@@ -0,0 +1,43 @@
+/*
+EXTRA_SOURCES: imports/issue18919b.d
+RUN_OUTPUT:
+---
+imports.issue18919b.func1: issue18919.d:27 issue18919.main void issue18919.main() issue18919
+imports.issue18919b.func2: issue18919.d:28 issue18919.main void issue18919.main() issue18919
+imports.issue18919b.func3: issue18919.d:29 issue18919.main void issue18919.main() issue18919
+imports.issue18919b.func3b: issue18919.d:30 issue18919.main void issue18919.main() issue18919
+imports.issue18919b.func3c: issue18919.d:31 issue18919.main void issue18919.main() issue18919
+imports.issue18919b.func3d: issue18919.d:32 issue18919.main void issue18919.main() issue18919
+imports.issue18919b.func4: issue18919.d:33 issue18919.main void issue18919.main() issue18919
+imports.issue18919b.func4b: issue18919.d:34 issue18919.main void issue18919.main() issue18919
+imports.issue18919b.func4c: issue18919.d:35 issue18919.main void issue18919.main() issue18919
+imports.issue18919b.func4d: issue18919.d:36 issue18919.main void issue18919.main() issue18919
+imports.issue18919b.func5!("issue18919.d", 37, "issue18919.main", "void issue18919.main()", "issue18919").func5: issue18919.d:37 issue18919.main void issue18919.main() issue18919
+imports.issue18919b.func6!("issue18919.d", 38, "issue18919.main", "void issue18919.main()", "issue18919").func6: issue18919.d:38 issue18919.main void issue18919.main() issue18919
+imports.issue18919b.func7: expr1=1078, file=issue18919.d func=issue18919.main, expr3=1
+imports.issue18919b.func8: expr1=[40, 1040, ], expr2=[issue18919.d: 40, ], expr3=constant2, expr4=issue18919
+imports.issue18919b.func9.fp: issue18919b.d:216 imports.issue18919b
+imports.issue18919b.func10: expr1=imports.issue18919b, expr2=imports.issue18919b
+---
+*/
+import imports.issue18919b;
+
+void main()
+{
```

Comment:
Could this be a `compilable`? (They're much faster).

Reply:
No, the test checks the output of the program at runtime und `RUN_OUTPUT`.

Reply:
A lot of the compilables use a static assert to check values.

Reply:
The values are not always available at compile time. Only `func5` and `func6` pass the values as template parameters, so they could be checked with static assert. Most of the test functions pass the values using runtime parameters, so a test at runtime is still necessary.

Reply:
Maybe it would be possible to run everything at CTFE. Then it could be in `compilable`. But then it would not test that for example `__FILE__.ptr` is correctly handled by the backend.